tests/migrations/test_optimizer.py

```diff
@@ -631,3 +632,34 @@ def test_optimize_through_fields(self):
                 migrations.CreateModel("Bar", [("width", models.IntegerField())]),
             ],
         )
+
+    def test_optimize_elidable_operation(self):
+        self.assertOptimizesTo(
+            [
+                operations.special.ElidableOperation(elidable=True),
```

Comment:
Exactly, but shouldn't the outcome of the two be the same, regardless of the first operation being a `ElidableOperation` or not? I think the logic is a bit the other way round. The reduction is implemented but not the optimizeable through.